src/utilities/utilities.js

```diff
@@ -169,3 +170,44 @@ export const handleSelfFocus = e => {
     e.currentTarget.focus();
   }
 };
+
+export const isFullyVisible = (slideIndex, config) => {
+  const {
+    currentSlide,
+    cellSpacing,
+    slideCount,
+    slideWidth,
+    frameWidth,
+    wrapAround
+  } = config;
+
+  // Slide width can't be 0
+  const fullSlideWidth = slideWidth + cellSpacing || 1;
```

Comment:
Added this so tests would pass because most of our tests don't actually set a slideWidth (because the virtual dom doesn't have one).